timemachine/potentials/bonded.py

```diff
@@ -1,18 +1,26 @@
 import jax.numpy as np
 
-def centroid_restraint(conf, params, box, lamb, masses, group_a_idxs, group_b_idxs, kb, b0):
+def centroid_restraint(conf, params, box, lamb, group_a_idxs, group_b_idxs, kb, b0):
```

Comment:
Note `params`, `box`, `lamb` also unused -- can these arguments be safely omitted here, or is it important for these to be retained (e.g. so that the signature partially matches that of `harmonic_bond`)?

Reply:
Ahh! Intended as documented here: https://github.com/proteneer/timemachine/pull/382/files#diff-d0b9851a09ca44d39e9ef2f006a931da3a68103d2a2781370f5157cb05ee4038R88-R95 . Marking resolved